modules/asset-cdn/asset-cdn.php

```diff
@@ -0,0 +1,227 @@
+<?php
+/**
+ * Plugin Name: Asset CDN
+ * Description: Speed up Javascript and CSS
+ * Plugin URI: https://github.com/automattic/jetpack
+ * Author: Automattic
+ * Author URI: https://automattic.com
+ * Version: 0.1.0
+ * Text Domain: asset-cdn
+ * Domain Path: /languages/
+ * License: GPLv2 or later
+ */
+
+/**
+ * TODO
+ * - versioning (combine ver hashes) and cachebusting
+ * - concat/minify/serve JS too
+ * - asset inlining for smaller styles?
+ * - critical CSS support?
+ * - non-enqueued assets?
+ */
+
+class Asset_CDN {
+	private static $__instance = null;
+
+	private $cdn_server;
+	private $concat_style_groups = array();
+	private $concat_script_groups = array();
+	private $inject_critical_css = false;
+
+	/**
+	 * Singleton implementation
+	 *
+	 * @return object
+	 */
+	public static function instance() {
+		if ( ! is_a( self::$__instance, 'Asset_CDN' ) ) {
+			self::$__instance = new Asset_CDN();
+		}
+
+		return self::$__instance;
+	}
+
+	private function __construct() {
+		// $this->cdn_server = 'https://cdn.wpvm.io';
+		$this->cdn_server = 'http://localhost:8090';
```

Comment:
Since we're not setting this to anything in the constructor, maybe it should just be a constant?

Reply:
Made it filtered instead